changelogs/fragments/44-fixes-inv-script-host-option.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - Fail cleaner on invalid ``HOST`` arg to inventory script ``--host`` option (https://github.com/ansible-collections/community.digitalocean/pull/44).
+  - Return JSON consistent with spec from inventory script with ``--host`` (https://github.com/ansible-collections/community.digitalocean/pull/44).
```

Comment:
```suggestion
  - digital_ocean inventory script - return JSON consistent with specification with ``--host`` (https://github.com/ansible-collections/community.digitalocean/pull/44).
```

Reply:
@felixfontein Of course! Forgot about this, thanks!